src/main/scala/nl/knaw/dans/easy/stage/Settings.scala

```diff
@@ -47,8 +47,6 @@ object Settings {
              urn: String,
              doi: String,
              otherAccessDoi: Boolean,
-             stageFileDataAsRedirectDatastreams: Boolean,
-             fileDataRedirectBaseUrl: Option[URL],
```

Comment:
As we will merge staging together with ingest-flow, we no longer need a backward compatible API.